document-en.md

```diff
@@ -0,0 +1,244 @@
+    </tr>
+    <tr>
+        <td>Build error</td>
+        <td>Total number of error build.Build status will be changed to error when at least one commmand fail.</td>
+    </tr>
+    <tr>
+        <td>Build killed</td>
+        <td>Total number of killed build. Build status will be changed to killed if there's no logs show up in a period of time</td>
+    </tr>
+    <tr>
+        <td>Build skipped</td>
+        <td>Total number of skipped build. If user force push in process of build. Build's status will be changed to skipped</td>
+    </tr>
+    <tr>
+        <td>Build timeout</td>
+        <td>Total number of build exceed time, status will be changed to build timeout</td>
+    </tr>
+    <tr>
+        <td>Build running</td>
+        <td>Total number of running build.</td>
+    </tr>
+    <tr>
+        <td>Total User</td>
+        <td>Total number of user registered</td>
+    </tr>
+    <tr>
+        <td>2</td>
+        <td>Search</td>
+        <td>Filling project field</td>
+        <td>Search project by project's name</td>
+    </tr>
+    <tr>
+        <td>3</td>
+        <td>Repository/project list</td>
+        <td></td>
+        <td></td>
+    </tr>
+</table>
+
+## 3. Available repositories
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_012.png)
+ 
+ <table>
+    <tr>
+        <td>#</td>
+        <td>Description</td>
+    </tr>
+    <tr>
+        <td>1</td>
+        <td>This button update the repository list</td>
+    </tr>
+    <tr>
+        <td>2</td>
+        <td>Filter project by it's status ( All/Personal/Organization )</td>
+    </tr>
+    <tr>
+        <td>3</td>
+        <td>Search project by name</td>
+    </tr>
+    <tr>
+        <td>4</td>
+        <td>Repository/Project list fetched from github account </td>
+    </tr>
+</table>
+
+> You need to be an admin of repository to active project and create hook on github
+> 
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_013.png)
+
+> 
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_014.png)
+
+## 4. Overview
+Overview page display sumary information, build statistic of repository.This page show up when you click on a project actived at homepage.
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_015.png)
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_016.png)
+
+>1. The last build information
+>2. Number of builds, success build and error build 
+>3. This chart display errors of 10 recently builds. 
+>4. This chart analyze builds in 10 days. 
+## 5. Build
+By click on each build, detail page will show up, detail page include some detail as below:
+> Logs: Display log result of each command that you defined in config file.
+
+> Coverage: Unit test result
+
+> Violations: Detail information of code convention violated in project.
```

Comment:
`Detail information of code convention violated in project` 
=> 
`Detail information of code conventions violated in the project`